server/README.md

```diff
@@ -0,0 +1,38 @@
+# Hasal Online Server
+
+## Running
+
+By default, the config files will put under `~/.hasal_server` folder.
```

Comment:
By default, the config files `are` under ~/.hasal_server folder.
or
By default, the config files `should be put' under ~/.hasal_server folder.